src/Service/SpamCheckService.php

```diff
@@ -33,39 +42,41 @@ public function __construct($apiKey, $blog)
      */
     public function isCommentAcceptable(array $data, $userIp, $userAgent)
     {
-        if (!array_key_exists('comment', $data) || !is_string($data['comment']) || '' === trim($data['comment'])) {
+        try {
```

Comment:
@localheinz Can you double-check my work here in the conflict resolution is correct?